src/Math-Matrix/PMMatrix.class.st

```diff
@@ -520,7 +521,8 @@ PMMatrix >> initializeRows: anArrayOrVector [
 { #category : #initialization }
 PMMatrix >> initializeRows: rowsInteger columns: columnsInteger [
 	"Build empty components for a matrix."
-		
+	[ rowsInteger  isInteger and: [ rowsInteger   >= 0 ] ] assert.
+	[ columnsInteger  isInteger and: [ columnsInteger  >= 0 ] ] assert.
```

Comment:
What failure message do clients get here? I'd it easy to debug from?

Reply:
It shows up as an assertion failure only.

Reply:
Would you mind posting it here? It might be worth documenting that message in the test as an alternative.

Reply:
`AssertionFailure: Assertion failed`

Reply:
Perhaps, then, we should be clearer e.g. 'Dimension of a matrix must be greater than zero'